src/openms/source/FORMAT/MzQCFile.cpp

```diff
@@ -54,14 +57,15 @@ using namespace std;
 
 namespace OpenMS
 { 
-
   void MzQCFile::store(const String& input_file,
                        const String& output_file,
                        const MSExperiment& exp,
                        const String& contact_name,
                        const String& contact_address,
                        const String& description,
-                       const String& label) const
+                       const String& label,
+                       const String& inputfile_feature,
```

Comment:
could we just pass a const&FeatureMap and adapt the code that we don't need to read the featuremap in the QcML class? That way we could use it for featuremaps that are in memory